request/request.go

```diff
@@ -1,65 +1,148 @@
+/*
+A set of functions to retrieve information about the incoming requests made by clients.
+*/
 package request
 
 import (
 	"github.com/Kong/go-pdk/bridge"
 )
 
+// Holds this module's functions.  Accessible as `kong.Request`
 type Request struct {
 	bridge.PdkBridge
 }
 
+// Called by the plugin server at initialization.
 func New(ch chan interface{}) Request {
 	return Request{bridge.New(ch)}
 }
 
+// kong.Request.GetScheme() returns the scheme component of the request’s URL.
+// The returned value is normalized to lower-case form.
 func (r Request) GetScheme() (s string, err error) {
 	return r.AskString(`kong.request.get_scheme`)
 }
 
+// kong.Request.GetHost() returns the host component of the request’s URL,
+// or the value of the “Host” header. The returned value is normalized
+// to lower-case form.
 func (r Request) GetHost() (host string, err error) {
 	return r.AskString(`kong.request.get_host`)
 }
 
+// kong.Request.GetPort() returns the port component of the request’s URL.
```

Comment:
Lua docs mention that this returns a number, why is it a string here?

Reply:
Looks like it's just a bug in the go-pdk? Is there a `AskNumber` function?